README.md

```diff
@@ -15,3 +15,40 @@ To run the site locally with Docker, use the following command:
 ```bash
 docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll jekyll serve
 ```
+
+
+
+#### Side note for those on non-linux machines
+ 
```

Comment:
Yah, docker on mac works great. But the version of minikube i was using (0.17.1) when combined with virtual box didn't quite work well. Using xhyve (which is IMHO way better than virtual box .. for either minikube OR docker for mac) worked better for Minikube. I'll update the PR to clarify that.